core/src/main/scala/org/apache/spark/internal/config/package.scala

```diff
@@ -429,7 +429,11 @@ package object config {
         "external shuffle service, this feature can only be worked when external shuffle" +
         "service is newer than Spark 2.2.")
       .bytesConf(ByteUnit.BYTE)
-      .createWithDefault(Long.MaxValue)
```

Comment:
I think the original purpose to set to `Long.MaxValue` is to avoid using this configuration by default, user should set to a proper size to enable this feature. But anyway I think the current change is also fine.